content/en/docs/reference/config/telemetry/metrics/index.md

```diff
@@ -6,7 +6,7 @@ weight: 50
 
 The following are the standard service level metrics exported by Istio.
 Istio standard metrics are directly exported by the Envoy proxy since Istio 1.5.
-The telemetry component is [implemented](https://github.com/istio/proxy/tree/release-1.5/extensions/stats) as a [Proxy-wasm](https://github.com/proxy-wasm/spec) plugin.
+The telemetry component is [implemented](https://github.com/istio/proxy/tree/master/extensions/stats) as a [Proxy-wasm](https://github.com/proxy-wasm/spec) plugin.
```

Comment:
@kyessenov Does this the doc change need to be cherrypicked into the release-1.5 branch? If so, is pointing to the master branch of istio/proxy right?